recipes/basenji/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+export C_INCLUDE_PATH=$PREFIX/include
```

Comment:
Why is this needed? There is no compiler here or any libraries.

Reply:
Good question, probably a copy/paste accident when I started this recipe in 2020, removed it :)